test/functional/p2p_tx_download.py

```diff
@@ -274,6 +274,7 @@ def test_preferred_tiebreaker_inv(self):
         pref_peer = self.nodes[0].add_outbound_p2p_connection(
            TestP2PConn(), wait_for_verack=True, p2p_idx=1, connection_type="outbound-full-relay")
         pref_peer.send_message(msg_inv([CInv(t=MSG_WTX, h=0xff00ff00)]))
+        pref_peer.sync_with_ping()
```

Comment:
could also `send_and_ping`

Reply:
seems to cause quite a few nuisance issues in tests, may be worth it going forward